src/views/compute/DeployVM.vue

```diff
@@ -89,12 +89,18 @@
                           :selected="tabKey"
                           :loading="loading.templates"
                           :preFillContent="dataPreFill"
-                          @update-template-iso="updateFieldValue"
-                        ></template-iso-selection>
+                          @update-template-iso="updateFieldValue" />
+                        <span>
+                          {{ $t('Override root disk size') }}
```

Comment:
@davidjumani Can you fix translation/key label usage across views (see locales/en.json to pick the keys, all lowercase)

Reply:
Done!